code/index.html

```diff
@@ -1,22 +1,109 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap" rel="stylesheet">
+    <title>Sunrise Travel</title>
+</head>
+
+<body>
+
     <header>
-      <!-- Some kind of logo and navbar -->
+        <!-- logo and navbar -->
+        <img src="./pics/logo.png" alt="logo">
+
+        <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <nav>
+            <a href="#">Contact</a>
+            <a href="#">Destinations</a>
+            <a href="#">Booking</a>
+        </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+
+    <section class="info">
+        <!-- <div class="image"> -->
+        <img src="pics/fiji.jpg" alt="vaccination image">
+        <!-- </div> -->
+        <div class="text">
+            <h1>Find deals on hotels, flights <br>and package trips.</h1>
+            <h3><i>We have destinations all over the world.</i></h3>
```

Comment:
Same thing here with the `<i>`-tag, use the `font-style: italic;` property in CSS instead 🤗